drivers/watchdog/wdt_nrfx.c

```diff
@@ -6,6 +6,7 @@
 
 #include <nrfx_wdt.h>
 #include <drivers/watchdog.h>
+#include <devicetree.h>
```

Comment:
Is it needed? We don't do it in other drivers.

Reply:
I can remove it; drivers/watchdog.h pulls in device.h, which pulls in kernel.h, which ends up pulling in devicetree.h via our arch cpu.h. It's probably safe to assume that will continue to be true.